lib/src/GlobalFilters.cc

```diff
@@ -0,0 +1,86 @@
+#include <drogon/plugins/GlobalFilters.h>
+#include <drogon/DrClassMap.h>
+#include <drogon/HttpAppFramework.h>
+#include "FiltersFunction.h"
+#include "HttpRequestImpl.h"
+
+using namespace drogon::plugin;
+
+void GlobalFilters::initAndStart(const Json::Value &config)
+{
+    if (config.isMember("filters") && config["filters"].isArray())
+    {
+        auto &filters = config["filters"];
+
+        for (auto const &filter : filters)
+        {
+            if (filter.isString())
+            {
+                auto filterPtr = std::dynamic_pointer_cast<HttpFilterBase>(
+                    drogon::DrClassMap::getSingleInstance(filter.asString()));
+                if (filterPtr)
+                {
+                    filters_.push_back(filterPtr);
+                }
+                else
+                {
+                    LOG_ERROR << "Filter " << filter.asString()
+                              << " not found!";
+                }
+            }
+        }
+    }
+    if (config.isMember("exempt"))
+    {
+        auto exempt = config["exempt"];
+        if (exempt.isArray())
+        {
+            std::string regexStr;
+            for (auto const &ex : exempt)
+            {
+                if (ex.isString())
+                {
+                    regexStr += ex.asString() + "|";
+                }
```

Comment:
I think we need to validate `ex` being a valid regex before string append. Otherwise `regexStr` can be valid but the behavior is not what's expected. Ex `"exampt": ["/api(". "/api/)"]` will end up with `/api(|/api)`. Which is not what we are expecting.